packages/integrations/src/offline.ts

```diff
@@ -109,7 +109,7 @@ export class Offline implements Integration {
    * @param event an event
    */
   private async _cacheEvent(event: Event): Promise<Event> {
-    return this.offlineEventStore.setItem<Event>(uuid4(), normalize(event));
+    return this.offlineEventStore.setItem<Event>(uuid4(), normalize(event) as Event);
```

Comment:
> it would be more correct

It's a good point, and you're not wrong. Something to think about in whatever type-fixing PR we might do as a follow-up (see my comment above).